agent/security_helper.py

```diff
@@ -83,38 +83,45 @@ def is_app_armor_enabled():
     Returns a True/False if AppArmor is enabled.
     """
     try:
-        from sh import aa_status
+        import LibAppArmor
+        return LibAppArmor.aa_is_enabled() == 1
     except ImportError:
-        return False
+        try:
+            from sh import aa_status
+        except ImportError:
```

Comment:
pls comment what all execution branches mean

Reply:
nope. e.g. it's not clear what does it mean if `LibAppArmor` isn't available for import

Reply:
ok I'll add a comment